src/methods/get-function-info.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Retrieve function information from the Shopify CLI
+ */
+
+import { execSync } from 'child_process';
+
+/**
+ * Information about a Shopify function
+ */
+export interface FunctionInfo {
+  schemaPath: string;
+  functionRunnerPath: string;
+  wasmPath: string;
+  targeting: Record<string, any>;
+}
+
+/**
+ * Retrieves function information from the Shopify CLI
+ * @param {string} functionDir - The directory path of the function
+ * @returns {FunctionInfo} Function information including schemaPath, functionRunnerPath, wasmPath, and targeting
+ * @throws {Error} If the CLI command is not available or fails
+ */
+export function getFunctionInfo(functionDir: string): FunctionInfo {
+  let functionInfoJson: string;
+
+  try {
+    functionInfoJson = execSync(
+      `shopify app function info --json --path ${functionDir}`,
+      {
+        encoding: 'utf-8'
+      }
+    );
```

Comment:
Any reason we are using `execSync` here over `spawn` (as we do in other commands, such as the previous runFunction and buildFunction) ?

I'm not sure what the difference is, just noticed it wasn't the same.